src/components/CommentSection.jsx

```diff
@@ -53,6 +54,25 @@ export default function CommentSection({ article_id }) {
     setComment(e.target.value);
   };
 
+  const removeComment = (comment_id) => {
+    deleteComment(comment_id)
+      .then(() => {
+        setComments(comments.filter((comment) => comment.comment_id !== comment_id));
+        setMessage("Comment deleted successfully");
+      })
+      .catch((err) => {
+        console.error("Error deleting comment:", err);
+        setError("Failed to delete comment");
```

Comment:
This error causes all the comments to dissappear from the page when it happens. Eg trying to delete a comment when offline.